### PR TITLE
Screens 1 - UI Updates

### PR DESCRIPTION
**Pull request by ngvtuan**
##### Commits
- [x] fixed ui for welcome screen
- [x] fixed tooltip
- [x] fixed ui for onboarding step 1, step 2, step 3, step 4
- [x] added download pdf file
- [x] added download QR code video
- [x] updated ui password recovery screen
- [x] add generate password
- [x] added animation close icon
- [x] added links for buttons
- [x] updated ui for set new password

##### FreedCamp
- [x] [Screens 1 - UI Updates](https://freedcamp.com/view/2956183/tasks/41426263)

### CODE REVIEW

**Review by CrazyOptimist — DISMISSED** [review]
Looks good, thanks for refactoring `href` to `to`

File: src/features/onboarding/register/StepBackup.tsx [hunk 1]
```diff
@@ -15,17 +26,125 @@ export type TStepBackupMethodProps = {
...
+        password,
+        url,
+      }
+      const results = await fetchPastelIDAndPrivateKeys(rpcConfig)
+      if (results) {
+        const chunks = splitStringIntoChunks(results, chunkQuantity)
+        setQRcodeData(chunks)
+      }
+
+      const keys = await fetchAllKeysForPdf(rpcConfig)
```
[the remark this answers is not in the document]
Comment (ngvtuan) [reply src/features/onboarding/register/StepBackup.tsx]: Updated

**Comment by sanjanadhikari1** [comment]
image
I think input box looks a little different with design.

**Comment by sanjanadhikari1** [comment]
image

**Comment by sanjanadhikari1** [comment]
image

**Comment by sanjanadhikari1** [comment]
image

**Review by sanjanadhikari1 — APPROVED** [review]
It looks good to me.

**Review by CrazyOptimist — APPROVED** [review]

**Pull request merged**